src/account.ts

```diff
@@ -140,6 +140,13 @@ export class Account {
 
         const flatLogs = [result.transaction_outcome, ...result.receipts_outcome].reduce((acc, it) => acc.concat(it.outcome.logs), []);
         this.printLogs(signedTx.transaction.receiverId, flatLogs);
+        // Find all failures in receipts, print out all except last which is handled later.
+        const receiptFailures = result.receipts_outcome.filter(o => typeof o.outcome.status === 'object' && typeof o.outcome.status.Failure === 'object').map(o => o.outcome.status.Failure);
```

Comment:
nit: seems like can be just `.filter({ outcome } => outcome.status && outcome.status.Failure)`